8/mingw32/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM fedora:39
```

Comment:
Can wet update this to Fedora 41 while we're at it?

Reply:
sure, can try